pages/_app.tsx

```diff
@@ -13,12 +20,18 @@ import Analytics from "@/components/Analytics";
 import Toast from "@/components/Toast";
 import { ModalContext } from "@/lib/context";
 import { GloSequenceConnector } from "@/lib/sequence-connector";
-import { getAllowedChains } from "@/lib/utils";
+
+import { isProd, isE2E } from "../lib/utils";
 
 import type { AppProps } from "next/app";
 
 const { chains, publicClient, webSocketPublicClient } = configureChains(
-  getAllowedChains(),
+  // Add conditional flag if IS_E2E_TEST? So we have polygon avialable as front one.
```

Comment:
remove comment

side note: I'd suggest starting with isProd as it's most crucial path hence it's better to have it in beginning